src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1009,6 +1045,95 @@ public void UpdateReportTabStatistics(Patch patch)
         OnPhysicalConditionsChartLegendPressed("temperature");
     }
 
+    public void UpdateTimeline()
+    {
+        var mainContainer = timelineSubtab.GetNode<Control>(
+            "MarginContainer/ScrollContainer/MarginContainer/VBoxContainer");
```

Comment:
Why not use an Export path here? This seems a very long path that is bound to get changed due to editor GUI changes at some point.